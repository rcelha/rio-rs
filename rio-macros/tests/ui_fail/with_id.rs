use rio_macros::*;

#[derive(Default, WithId)]
struct StructWithId {
    name: String,
}

fn main() {}
