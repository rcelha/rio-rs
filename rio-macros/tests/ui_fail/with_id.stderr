error: proc-macro derive panicked
 --> tests/ui_fail/with_id.rs:3:19
  |
3 | #[derive(Default, WithId)]
  |                   ^^^^^^
  |
  = help: message: StructWithId doesn't have an `id` attribute of type `String`
